Makefile

```diff
@@ -42,8 +47,8 @@ test-complete: test-env-clean test-env-down remove-tf-state test-env-up test-run
 
 .PHONY: remove-tf-state
 remove-tf-state:
-	@rm -rf ./modules/k3scluster/.terraform
-	@rm -rf ./modules/k3scluster/.terraform.lock.hcl ./modules/k3scluster/terraform.tfstate ./modules/k3scluster/terraform.tfstate.backup
+	@rm -rf ./modules/rke2/.terraform
+	@rm -rf ./modules/rke2/.terraform.lock.hcl ./modules/rk2/terraform.tfstate ./modules/rke2/terraform.tfstate.backup
 
```

Comment:
since for now we still have both products in model dir , we should or have to make commands like :

remove-tf-state-rke2
remove-tf-state-k3s

or if you want one command we should pass the product as arg, and based on that go to the specific dir.

Reply:
Yeah I vote for one command, will update

Reply:
Updated